admin/acceptance.py

```diff
@@ -794,6 +811,20 @@ def postOptions(self):
                 provider_config=provider_config,
             )
 
+    def _make_cluster_identity(self, dataset_backend):
+        """
+        Build a cluster identity based on the parameters.
+        """
+        cluster_id = make_cluster_id(
+            TestTypes.ACCEPTANCE,
+            _provider_for_cluster_id(dataset_backend),
+        )
+        return ClusterIdentity(
+            purpose=u'acceptance-testing',
```

Comment:
The current script uses `acceptance-test-` as the prefix for the nodes, but `acceptance-testing` for the metadata purpose. Add a `prefix` field to the ClusterIdentity and set it to `acceptance-test` here.  Use `prefix` when generating the node name.

In `cluster_setup.py`, `prefix` can be set to the same value as `purpose`.